src/components/SettingsPage/SyncCard.tsx

```diff
@@ -152,59 +155,77 @@ export const SyncCard = () => {
   }, [updateLastEntitySyncTime]);
 
   return (
-    <InfoCard
-      title="Sync entities"
-      action={
-        <SyncButton
-          isSyncing={syncTaskProgressPercentage !== null || isSubmittingTask}
-          submitSyncTask={submitEntitySync}
-        />
-      }
-    >
-      {syncTaskProgressPercentage !== null && (
-        <Grid
-          container
-          alignItems="center"
-          direction={'row'}
-          justifyContent="center"
-          style={{ marginBottom: '2px' }}
-        >
-          <Grid item lg={10} data-testid={`PollingLinearGauge-entity-sync`}>
-            <PollingLinearGauge
-              done={false}
-              poll={updateEntitySyncProgress}
-              value={syncTaskProgressPercentage}
+    <>
+      {cortexSyncError !== undefined && (
+        <WarningPanel severity="error" title={cortexSyncError}>
+          There is already a sync in progress. Try again in 10 minutes.
+        </WarningPanel>
+      )}
+      <InfoCard
+        title="Sync entities"
+        action={
+          cortexSyncError === undefined && (
+            <SyncButton
+              isSyncing={
+                syncTaskProgressPercentage !== null || isSubmittingTask
+              }
+              submitSyncTask={async () => {
+                try {
+                  await submitEntitySync();
+                  setCortexSyncError(undefined);
```

Comment:
We should probably clear the error on re-submit instead of re-success
```suggestion
                  setCortexSyncError(undefined);
                  await submitEntitySync();
```